gateways/istio-ingress/values.yaml

```diff
@@ -109,4 +109,23 @@ gateways:
       # enable cross cluster routing.
       ISTIO_META_ROUTER_MODE: "sni-dnat"
 
-
+    # Specify the pod anti-affinity that allows you to constrain which nodes
+    # your pod is eligible to be scheduled based on labels on pods that are
+    # already running on the node rather than based on labels on nodes.
+    # There are currently two types of anti-affinity:
+    #    "requiredDuringSchedulingIgnoredDuringExecution"
+    #    "preferredDuringSchedulingIgnoredDuringExecution"
+    # which denote “hard” vs. “soft” requirements, you can define your values
+    # in "podAntiAffinityLabelSelector" and "podAntiAffinityTermLabelSelector"
+    # correspondingly.
+    # For example:
+    # podAntiAffinityLabelSelector:
+    # - key: security
+    #   operator: In
+    #   values: S1,S2
+    #   topologyKey: "kubernetes.io/hostname"
+    # This pod anti-affinity rule says that the pod requires not to be scheduled
+    # onto a node if that node is already running a pod with label having key
+    # “security” and value “S1”.
+    podAntiAffinityLabelSelector: {}
```

Comment:
What is the effect of empty selector - will it trigger the if ? Don't remember of {{ if {} }} is true of false...

Reply:
Will not trigger if for empty selector.